app/controllers/widgets_controller.rb

```diff
@@ -19,6 +20,8 @@ def new
 
   # GET /widgets/1/edit
   def edit
+    x = %w(a b c)
+    x.each{|f| "hello"}
```

Comment:
Unused block argument - `f`. You can omit the argument if you don't care about it.